game/world/triggers/abstracttrigger.h

```diff
@@ -41,6 +42,7 @@ class TriggerEvent final {
       zenkit::MoverMessageType msg = zenkit::MoverMessageType::FIXED_DIRECT;
       int32_t                  key = 0;
       } move;
+    mutable AbstractTrigger* trigger = nullptr;
```

Comment:
OK, I see what you meant by "has to reference trigger directly instead of name".
`mutable` is definitely not a keyword we would like to use in game code.

One solution I've considered, is to track deferred trigger-events inside `AbstractTrigger`, and enable ticks as long as that list is non-empty